addons/web/static/tests/core/record_selectors/multi_record_selector.test.js

```diff
@@ -239,3 +264,30 @@ test("Backspace do nothing when the input is currently edited", async () => {
     await animationFrame();
     expect(".o_tag").toHaveCount(2);
 });
+
+// Desktop only because .o_data_row is only available on desktop
+test.tags("destop")("Can pass domain to search more", async () => {
+    MorePartner._views["list,false"] = /* xml */ `<tree><field name="name"/></tree>`;
```

Comment:
```suggestion
    Partner._records.push({ id: 4, name: "David" });
    // ...
    Partner._records.push({ id: 9, name: "Ivy" });
    // Required for the select create dialog
    class Users extends models.Model {
        _name = "res.users";
        has_group = () => true;
    }
    Partner._views["list,false"] = /* xml */ `<tree><field name="name"/></tree>`;
```
and remove all changes from above